packages/wdio-spec-reporter/src/types.ts

```diff
@@ -14,6 +14,7 @@ export interface Symbols {
 }
 
 export interface SpecReporterOptions {
+    sauceLabsSharableLinks?: boolean
```

Comment:
can we add some docs in here, similar to e.g. https://github.com/webdriverio/webdriverio/blob/main/packages/wdio-junit-reporter/src/types.ts

Reply:
Done